### DISCUSSION TITLE
I have a Query that uses offset-style pagination; how to approach?

### DISCUSSION THREAD

**Discussion by shaunatparallel** [profusion/sgqlc#208] [Q&A]
I have a Query that takes offset and returns 10 of the object at a given time. Is there a way to use Relay or some other pagniation approach to grab this?

The simple way would just to be a for loop and update the offset number every iteration.

**Comment by barbieri** [comment]
yes, you need to loop, see https://github.com/profusion/sgqlc/blob/master/examples/github/github_agile_dashboard.py#L203-L206

**Reply by shaunatparallel** [reply]
@barbieri thanks for the response. I guess I don't understand how to overlay the Relay approach over the offset. 

My Query 

`
itemQuery
(
offset: [Int] # Pass in the number of the record that you want to start retieving from. 
) {
name 
}
`

'
The above query just returns 10 items starting from the offset that is passed in. No nodes/edges or a cursor.
